snakebite/client.py

```diff
@@ -393,6 +394,60 @@ def _handle_rename(self, path, node, dst):
         response = self.service.rename(request)
         return {"path": path, "result": response.result}
 
+
+    def rename_dont_move(self, path, dst, overwriteDest=False):
+        ''' Rename (but don't move) path to a destination
+
+        Since you can't move with this operation, and only rename, it would
+        not make sense to pass multiple paths to rename to a single
+        destination. This method uses the underlying rename2 method.
+
+        https://github.com/apache/hadoop/blob/ae91b13/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/protocol/ClientProtocol.java#L483-L523
+
+        This method only wraps the FileAlreadyExistsException exception
+        (both for the Java FileAlreadyExistsException error and for "rename
+        destination directory is not empty" when overwriteDest=True). The other
+        exceptions will just be passed along.
+
+        :param path: Source path
+        :type path: string
+        :param dst: destination
+        :type dst: string
+        :returns: A dictionary or None
+        '''
+        if not path:
```

Comment:
Since path should be a single path - please add check for str/unicode, for example: "if not isinstance(path, (str, unicode)): destroy_the_world()"